README.md

```diff
@@ -1,19 +1,157 @@
-# Primitives for Alpaka
-This contains a `reduce` and a `transform` primitive for Alpaka, both header-only.
-Directories:
-* `include/vikunja`: The actual code
-    - `mem/iterator`: Contains a base iterator class and a policy based iterator that can vary the memory access pattern: Either a linear or a grid-striding access is possible.
-    - `reduce`: Contains the `transform_reduce` and the `reduce` function (the latter is just a convenient wrapper around the former).
-        + `detail`: This contains the two possible reduce kernels.
-    - `transform`: Contains two `transform` variants with one and two input iterators.
-        + `detail`: This contains the transform kernel.
-    - `workdiv`: This contains various working divisions for the different backends.
+# Vikunja
+
+[![Build Status](https://gitlab.com/hzdr/crp/vikunja/badges/master/pipeline.svg)](https://gitlab.com/hzdr/crp/vikunja/-/commits/master/)
+[![Documentation Status](https://readthedocs.org/projects/vikunja/badge/?version=latest)](https://vikunja.readthedocs.io)
+[![Doxygen](https://img.shields.io/badge/API-Doxygen-blue.svg)](https://vikunja.readthedocs.io/en/latest/doxygen/index.html)
+[![Language](https://img.shields.io/badge/language-C%2B%2B14-orange.svg)](https://isocpp.org/)
+[![Platforms](https://img.shields.io/badge/platform-linux-lightgrey.svg)](https://github.com/alpaka-group/vikunja)
+[![License](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](https://www.mozilla.org/en-US/MPL/2.0/)
+
+![vikunja](docs/source/logo/vikunja_logo.png)
+
+Vikunja is a performance portable algorithm library that defines functions operating on ranges of elements for a variety of purposes . It supports the execution on multi-core CPUs and various GPUs.
+
+Vikunja uses [alpaka](https://github.com/alpaka-group/alpaka) to implement platform-independent primitives such as `reduce` or `transform`. 
+
+# Getting Started
+
```

Comment:
This is a promising start. Maybe first put the `Install` section here which shows how to set everything up (`git clone` etc.).

Reply:
I thought it is more interesting to see the code first ;-) But installation first makes more sense, therefore I changed it.